Projects/Server.Tests/Tests/Maps/MapSelectionTests.cs

```diff
@@ -0,0 +1,138 @@
+using Xunit;
+using Server.Maps;
+using System.Collections.Generic;
+using Server.Json;
+using System.Text.Json.Serialization;
+using System.Text.Json;
+
+namespace Server.Tests.Tests.Maps
+{
+    public class MapSelectionTests
+    {
+        [Fact]
+        public void No_maps_enabled_on_construction()
```

Comment:
I don't have all of the rules on (analyzers are hard to configure properly, lots of conflicting stuff).
But it would definitely flag these function names as not being part of the style.
Let's use something like:
```cs
public void TestNoMapsEnabled()
{
}
```

Reply:
sure :+1: